build.gradle

```diff
@@ -112,6 +122,18 @@ dependencies {
 	} else {
 		compileOnly "top.theillusivec4.curios:curios-neoforge:${curios_version}"
 	}
+
+	if (project.runtime_include_cucumber.toBoolean()) {
+		localImplementation "com.blakebr0.cucumber:Cucumber:${cucumber_version}"
```

Comment:
You can put the cucumber dependency under the include mystical agriculture part and get rid of the runtime_include_cucumber variable